content/en/docs/how-tos/use-registries-in-build-farm.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Use Registries in Build Farm"
+date: 2020-12-15T16:08:01-04:00
+draft: false
+---
+
+The follow table shows the public DNS of each registry in build farm:
+
+| cluster | registry                                             | note                        |
+|---------|------------------------------------------------------|-----------------------------|
+| api.ci  | registry.svc.ci.openshift.org                        |                             |
+| app.ci  | registry.ci.openshift.org                            | the central CI registry     |
+| build01 | registry.build01.ci.openshift.org                    |                             |
+| build02 | registry.build02.ci.openshift.org                    |                             |
+| vsphere | registry.apps.build01-us-west-2.vmc.ci.openshift.org | only open to vsphere admins |
+
+# Login as a github user:
+
+* Ensure oc-cli has logged into the cluster: Copy the login command from [the console](/docs/getting-started/useful-links/#clusters).
+* Use oc token to log into the registry, e.g.,  `registry.ci.openshift.org`:
+
+```bash
+$ podman login -u $(oc whoami) -p $(oc whoami -t) --authfile ${HOME}/.docker/config.json registry.ci.openshift.org
```

Comment:
Sorry I didn't really understand the explanation for the `authfile`. So podman requires you to pass an `authfile` if you want to log into a registry? And why are you using that on a centos 8 machine?

Reply:
podman is always on a VM, never on the host of my laptop. easy to release the disk space by images, containers. Not really important here.

https://www.mankier.com/1/podman-login#Options---authfile=path

The point is that podman does not use `${HOME}/.docker/config.json` by default.
The default path is `/run/containers/0/auth.json`.

I want to show the content of the file in the `cat` command.
I feel it would be strange if `cat /run/containers/0/auth.json`.
So I explicitly set it up by `--authfile`.

However, `podman pull` command can read `${HOME}/.docker/config.json` without any issue so I removed that in the pull command below.

Let me replace podman with docker.
Less confusion.

Reply:
Updated.